sympy/series/tests/test_series.py

```diff
@@ -294,3 +294,20 @@ def test_issue_11407():
 
 def test_issue_14037():
     assert (sin(x**50)/x**51).series(x, n=0) == 1/x + O(1, x)
+
+
+def test_issue_12791():
+    beta = symbols('beta', reals=True, positive=True)
+    theta, varphi = symbols('theta varphi', reals=True)
+
+    expr = (-beta**2*varphi*sin(theta) + beta**2*cos(theta) + \
+        beta*varphi*sin(theta) - beta*cos(theta) - beta + 1)/(beta*cos(theta) - 1)**2
+
+    sol = 0.5/(0.5*cos(theta) - 1)**2 - 0.25*cos(theta)/(0.5*cos(theta) - 1)**2\
+        + (beta - 0.5)*(-0.5*varphi*sin(theta)*cos(theta)/((0.5*cos(theta) - 1)\
+        **2*(0.5*cos(theta) - 1.0)) - 1/(0.5*cos(theta) - 1)**2 + 0.5*cos(theta)\
+        **2/((0.5*cos(theta) - 1)**2*(0.5*cos(theta) - 1.0)) - 1.0*cos(theta)\
+        /((0.5*cos(theta) - 1)**2*(0.5*cos(theta) - 1.0))) + 0.25*varphi*\
+        sin(theta)/(0.5*cos(theta) - 1)**2 + O((beta - 0.5)**2, (beta, 0.5))
+
+    assert(expr.series(beta, 0.5, 2) == sol)
```

Comment:
I think this can be simply:
`assert expr.series(beta, 0.5, 2) == sol` (without round brackets).

Reply:
Ok, I'll make the changes suggested by you.